packages/create-react-native-library/templates/common/$.circleci/config.yml

```diff
@@ -71,6 +71,49 @@ jobs:
           path: coverage
           destination: coverage
 
+  e2e_release_ios:
+    executor:
+      name: rn/macos
+      xcode_version: '11.4.0'
+    steps:
+      - attach_workspace:
+          at: .
+      - rn/setup_macos_executor:
+          node_version: '12.10.0'
+      - rn/ios_simulator_start:
+          device: 'iPhone 11'
+      - rn/pod_install:
+          pod_install_directory: 'example/ios'
+      - run:
+          command: yarn --cwd example detox:ios:build:release
+          name: build for detox
+      - run:
+          command: detox clean-framework-cache && detox build-framework-cache && yarn --cwd example detox:ios:test:release
```

Comment:
Is the cache shared across different CI runs? If not, do these commands need to be run inside `example`?

Reply:
I don't remember, why I used this command, but it fixed something on circle CI. Should I remove it ?